tools/aggregate_team_overrides.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
```

Comment:
I would personally make a few minor changes for this like sending tracing / operations-related output to stderr rather than stdout (so that stdout can be the actual output of the aggregation and be used in common shell ways like `tool/aggregate_team_overrides.sh dir/ > output.yaml`), and maybe printing a more standard `usage: $0 <DIR>` style output on failures, and run the script through `shellcheck` to standardize style / avoid ambiguity. 

Practically speaking though, the above are super minor nits for a script that has a very specific purpose to be used as part of the update workflow. If you'd like me to expand on the above feedback or if you wanted to make some of those changes, feel free, but I think this is fine as-is.

Reply:
I can clean these up today, no problem.

Reply:
Ah nice, the script came out quite a bit simpler with those changes. :tada: